demo-app/src/main/java/org/openmined/syft/demo/ui/FederatedCycleViewModel.kt

```diff
@@ -114,6 +120,10 @@ class FederatedCycleViewModel(
         _processData.postValue(ProcessData(result))
     }
 
+    private fun postEpoch(epoch: Int) {
```

Comment:
This is not used anywhere. Maybe you intended to use it instead of `postLog("Starting epoch ${epoch + 1}")`?